kubemarine/templates/plugins/dashboard-v2.4.yaml.j2

```diff
@@ -236,10 +236,10 @@ spec:
           emptyDir: {}
       serviceAccountName: kubernetes-dashboard
       nodeSelector:
-        {{ plugins['kubernetes-dashboard']['dashboard']['nodeSelector'] | toyaml | indent(width=8, indentfirst=False) }}
+        {{ plugins['kubernetes-dashboard']['dashboard']['nodeSelector'] | toyaml | indent(width=8, first=False) }}
```

Comment:
jinja has some incompatible changes? It is necessary to understand that not only we use jinja, but Kubemarine users might potentially use the same or other jinja constructions and filters. So, incompatible changes in jinja == incompatible changes in Kubemarine. How we are going to announce that? The same question is for ansible in case if it also have incompatible changes.

Reply:
delete old file